src/scalper.ts

```diff
@@ -462,7 +463,11 @@ export class Scalper {
     } catch (err) {
         console.log(this.symbol, "Gamma Ask Error", err);
         console.log(this.symbol, "Gamma Ask Error Details",err.stack);
-      }
+    }
+    
+    // Sleep for the max time of the reruns then kill thread
+    await sleepExact((1 + percentDrift) * scalperWindow);
+    return;
```

Comment:
What is this supposed to do? The code should just exit normally already

Reply:
True will update to removeEventListeners that is the piece that lives on within a thread if we don't receive a fill